src/main/java/edu/cornell/kfs/concur/batch/service/impl/ConcurEventNotificationProcessingServiceImpl.java

```diff
@@ -0,0 +1,67 @@
+package edu.cornell.kfs.concur.batch.service.impl;
+
+import java.util.Collection;
+
+import org.apache.commons.collections.CollectionUtils;
+
+import edu.cornell.kfs.concur.batch.service.ConcurEventNotificationProcessingService;
+import edu.cornell.kfs.concur.businessobjects.ConcurAccountInfo;
+import edu.cornell.kfs.concur.businessobjects.ConcurEventNotification;
+import edu.cornell.kfs.concur.rest.xmlObjects.ExpenseDetailedReportDTO;
+import edu.cornell.kfs.concur.service.ConcurAccountValidationService;
+import edu.cornell.kfs.concur.service.ConcurEventNotificationService;
+
+public class ConcurEventNotificationProcessingServiceImpl implements ConcurEventNotificationProcessingService {
+    
+    protected ConcurEventNotificationService concurEventNotificationService;
+    protected ConcurAccountValidationService concurAccountValidationService;
+
+    @Override
+    public void processConcurEventNotifications() {
+        Collection<ConcurEventNotification> concurEventNotifications = concurEventNotificationService.retrieveConcurEventNotificationsForProcessing();
+
+        if (concurEventNotifications != null && CollectionUtils.isNotEmpty(concurEventNotifications)) {
+            for (ConcurEventNotification concurEventNotification : concurEventNotifications) {
+                ExpenseDetailedReportDTO expenseDetailedReport = retrieveExpenseDetailedReportFromConcur(concurEventNotification);
+                ConcurAccountInfo concurAccountInfo = extractAccountInfo(expenseDetailedReport);
+                concurAccountValidationService.validateConcurAccountInfo(concurAccountInfo);
+                updateExpenseReportStatusInConcur();
+            }
+        }
+    }
+
+    @Override
+    public ExpenseDetailedReportDTO retrieveExpenseDetailedReportFromConcur(
```

Comment:
I think I do have a template, but I'm using Intellij IDEA so I'm not sure it's sharable if you're using Eclipse. For reference, here are the Coding Standards - IDE Settings from the product team: https://kualico.atlassian.net/wiki/x/DgDrAg